ngi_visualizations/fpkm_scatter/fpkm_scatter.py

```diff
@@ -213,13 +255,98 @@ def plot_fpkm_scatter (sample_1, sample_2, x_lab, y_lab, output_fn=False, linear
     logging.info("Saving to {} and {}".format(png_fn, pdf_fn))
     plt.savefig(png_fn)
     plt.savefig(pdf_fn)
+    
+    # Return the filenames and R2
+    return {'png': png_fn, 'pdf': pdf_fn, output_fn:r_squared }
```

Comment:
I don't really understand.. Is there a reason that you can't use `'output_fn': r_squared` here?